app/controllers/api/v2/works_controller.rb

```diff
@@ -34,17 +34,17 @@ def create
     if status == :ok
       # Process the works, updating the flags
       works_responses = external_works.map { |external_work| import_work(archivist, external_work.merge(params.permit!)) }
-      success_works, error_works = works_responses.partition { |r| [:ok, :created, :found].include?(r[:status]) }
+      success_reponses, error_responses = works_responses.partition { |r| [:ok, :created, :found].include?(r[:status]) }
```

Comment:
This should be "success_responses".

Reply:
So it should. It wasn't the cause of the failing test, but I've sorted that out too, so this is ready to go!